components/sceneutil/riggeometry.cpp

```diff
@@ -355,7 +355,18 @@ void RigGeometry::setInfluenceMap(osg::ref_ptr<InfluenceMap> influenceMap)
             vec.emplace_back(std::make_pair(boneName, bi.mInvBindMatrix), weightPair.second);
         }
     }
-
+    
+    /// normalize weights 
+    for (auto& vertexPair : vertex2BoneMap)
```

Comment:
Do we really need a separate loop here? Why just to do not use a loop below?

Reply:
that's how normalization works

Reply:
an other method would be to deffer normalization to runtime and you don't want that

Reply:
perhaps you are right will see that tomorrow 'im to drunk right now

Reply:
i do that drunk....

Reply:
No, it's not resolved.

Reply:
So I should click the unresolve button?

Reply:
mp3butcher said something about how it doesn't work because BoneWeight is private but I'm not sure.

Reply:
@Capostrophic 
it's just that  can't use an inline function because of this....not a big deal